roocs_grids/scripts/retrieve_grids.sh

```diff
@@ -113,58 +102,54 @@ for nc in  ${CMIP6Grids[@]} ${ATLASGrids[@]}; do
+    ncdump -h "$targetpath/$nc" | grep -q "days since NA" && {
+        ncatted -O -a units,time,m,c,"days since 2001-01-01 00:00:00" "$targetpath/$nc" &&
+            echo "  ...successful for '$nc'." ||  echo "  ...failed for '$nc'!"
     }
 done
 echo -e "...done!"
 
-
-
 # Post Process ESGF Files
 echo -e "\nPost-processing ESGF files..."
-
 # Extract and merge variables
-[[ ! -e $targetpath/grid_T255.nc ]] && {
-    mv $targetpath/sfcWind_Amon_MPI-ESM1-2-XR_highresSST-present_r1i1p1f1_gn_195001-195012.nc $targetpath/grid_T255.nc
-    ncks -O -h -x -v sfcWind,time,time_bnds $targetpath/grid_T255.nc $targetpath/grid_T255.nc > /dev/null
-    nccopy -d 9 -s $targetpath/grid_T255.nc $targetpath/grid_T255_def.nc
-    mv $targetpath/grid_T255_def.nc $targetpath/grid_T255.nc
+[[ ! -e "$targetpath"/grid_T255.nc ]] && {
+    mv "$targetpath"/sfcWind_Amon_MPI-ESM1-2-XR_highresSST-present_r1i1p1f1_gn_195001-195012.nc "$targetpath"/grid_T255.nc
+    ncks -O -h -x -v sfcWind,time,time_bnds "$targetpath"/grid_T255.nc "$targetpath"/grid_T255.nc > /dev/null
+    nccopy -d 9 -s "$targetpath"/grid_T255.nc "$targetpath"/grid_T255_def.nc
+    mv "$targetpath"/grid_T255_def.nc "$targetpath"/grid_T255.nc
 }
-[[ ! -e $targetpath/grid_T127_lsm_binary.nc ]] && {
-    mv $targetpath/sftlf_fx_MPI-ESM1-2-HR_ssp370_r3i1p1f1_gn.nc $targetpath/grid_T127_lsm_binary.nc
-    ncks -A -h -v areacella $targetpath/areacella_fx_MPI-ESM1-2-HR_ssp245_r1i1p1f1_gn.nc $targetpath/grid_T127_lsm_binary.nc
-    nccopy -d 9 -s $targetpath/grid_T127_lsm_binary.nc $targetpath/grid_T127_lsm_binary_def.nc
-    mv $targetpath/grid_T127_lsm_binary_def.nc $targetpath/grid_T127_lsm_binary.nc
+[[ ! -e "$targetpath"/grid_T127_lsm_binary.nc ]] && {
+    mv "$targetpath"/sftlf_fx_MPI-ESM1-2-HR_ssp370_r3i1p1f1_gn.nc "$targetpath"/grid_T127_lsm_binary.nc
+    ncks -A -h -v areacella "$targetpath"/areacella_fx_MPI-ESM1-2-HR_ssp245_r1i1p1f1_gn.nc "$targetpath"/grid_T127_lsm_binary.nc
+    nccopy -d 9 -s "$targetpath"/grid_T127_lsm_binary.nc "$targetpath"/grid_T127_lsm_binary_def.nc
+    mv "$targetpath"/grid_T127_lsm_binary_def.nc "$targetpath"/grid_T127_lsm_binary.nc
 }
-[[ ! -e $targetpath/grid_T63_lsm_binary.nc ]] && {
-    mv $targetpath/sftlf_fx_MPI-ESM1-2-LR_1pctCO2_r1i1p1f1_gn.nc $targetpath/grid_T63_lsm_binary.nc
-    ncks -A -h -v areacella $targetpath/areacella_fx_MPI-ESM1-2-LR_1pctCO2_r1i1p1f1_gn.nc $targetpath/grid_T63_lsm_binary.nc
-    nccopy -d 9 -s $targetpath/grid_T63_lsm_binary.nc $targetpath/grid_T63_lsm_binary_def.nc
-    mv $targetpath/grid_T63_lsm_binary_def.nc $targetpath/grid_T63_lsm_binary.nc
+[[ ! -e "$targetpath"/grid_T63_lsm_binary.nc ]] && {
+    mv "$targetpath"/sftlf_fx_MPI-ESM1-2-LR_1pctCO2_r1i1p1f1_gn.nc "$targetpath"/grid_T63_lsm_binary.nc
+    ncks -A -h -v areacella $targetpath/areacella_fx_MPI-ESM1-2-LR_1pctCO2_r1i1p1f1_gn.nc "$targetpath"/grid_T63_lsm_binary.nc
+    nccopy -d 9 -s "$targetpath"/grid_T63_lsm_binary.nc "$targetpath"/grid_T63_lsm_binary_def.nc
```

Comment:
I will need to extend the compression to all downloaded grid files and remove unnecessary attributes, variables and dimensions. By just applying `nccopy -d 9 -s` to all the grid files, the volume could be shrinked from 45M to 2.4M, which would be fine? The volume was one of the reasons, `roocs_grids` was a separate repository and not part of the clisops regridding PR.

I will follow up your PR with another one dealing with the compression and further cleaning of the netcdf files.